docs/getting-started-guides/ubuntu/automated.md

```diff
@@ -0,0 +1,253 @@
+     - Three node Kubernetes cluster with one master and two worker nodes.
+     - TLS used for communication between nodes for security.
+     - Flannel Software Defined Network (SDN) plugin
+     - A load balancer for HA kubernetes-master (Experimental)
+     - Optional Ingress Controller (on worker)
+     - Optional Dashboard addon (on master) including Heapster for cluster monitoring
+- EasyRSA
+     - Performs the role of a certificate authority serving self signed certificates
+       to the requesting units of the cluster.
+- Etcd (distributed key value store)
+     - Three node cluster for reliability.
+- Elastic stack
+     - Two nodes for ElasticSearch
+     - One node for a Kibana dashboard
+     - Beats on every Kubernetes and Etcd node:
+          - Filebeat for forwarding logs to ElasticSearch
+          - Topbeat for inserting server monitoring data to ElasticSearch
+
+
+The Juju Kubernetes work is curated by a dedicated team of community members,
+let us know how we are doing. If you find any problems please open an
+[issue on the kubernetes project](https://github.com/kubernetes/kubernetes/issues)
+and tag the issue with "juju" so we can find them.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+- A working [Juju client](https://jujucharms.com/docs/2.0/getting-started-general); this does not have to be a Linux machine, it can also be Windows or OSX.
+- A [supported cloud](http://localhost:8090/#cloud-compatibility).
+
+### On Ubuntu
+
+On your local Ubuntu system:  
+
+```shell
+sudo add-apt-repository ppa:juju/stable
+sudo apt-get update
+sudo apt-get install juju
+```
+
+If you are using another distro/platform - please consult the
+[getting started guide](https://jujucharms.com/docs/2.0/getting-started-general)
+to install the Juju dependencies for your platform. 
+
+### Configure Juju to your favorite cloud provider
+
+Deployment of the cluster is [supported on a wide variety of public clouds](#cloud-compatibility), private OpenStack clouds, or raw bare metal clusters.  
+
+After deciding which cloud to deploy to, follow the [cloud setup page](https://jujucharms.com/docs/devel/getting-started-general#2.-choose-a-cloud) to configure deploying to that cloud. 
+
+Load your [cloud credentials](https://jujucharms.com/docs/2.0/credentials) for each 
+cloud provider you would like to use. 
+
+In this example
+
+```shell
+juju add-credential aws 
+credential name: my_credentials
+select auth-type [userpass, oauth, etc]: userpass
+enter username: jorge
+enter password: *******
+```
+
+You can also just auto load credentials for popular clouds with the `juju autoload-credentials` command, which will auto import your credentials from the default files and environment variables for each cloud. 
+
+Next we need to bootstrap a controller to manage the cluster. You need to define the cloud you want to bootstrap on, the region, and then any name for your controller node:   
+
+```shell
+juju bootstrap aws/us-east-2 example_cluster
```

Comment:
I don't think including `example_cluster` is required. In 2.0 it'll automatically name the controller